CONTRIBUTORS

```diff
@@ -48,7 +48,6 @@ Cristian Atehortua <cristian25a32@gmail.com>
 Damien Deis <developer.deis@gmail.com>
 Dany L'Hébreux <danylhebreux@gmail.com>
 Dave Nicholas <davenicholasuk@gmail.com>
-David Pfister <pfisterdd10@gmail.com>
```

Comment:
Even if PR is reversed, I am not in favor of eliminating people from here, because they have already contributed!